run-tests-with-build-kogito-decision-services-quickstarts-optawebs/README.md

```diff
@@ -0,0 +1,33 @@
+run-tests-with-build-kogito-decision-services-quickstarts-optawebs
+------------------------------------
+Run tests OptaWeb backend, frontend and standalone, including integration delivered as part of decision-services-quickstarts zip
+
+Currently, these optawebs are delivered in decision-services-quickstarts zip
+- [optaweb-employee-rostering](https://github.com/kiegroup/optaweb-employee-rostering)
+- [optaweb-vehicle-routing](https://github.com/kiegroup/optaweb-vehicle-routing)
+
+Execution
+-------------------------------------
+`mvn verify -Doptaweb=$OPTAWEB_NAME -s $PROJECT_SETTINGS -Dkogito.decision.services.quickstarts.url=$QUICKSTARTS_URL`
+
+where 
+
+`$OPTAWEB_NAME` is a profile name {vehicle-routing|employee-rostering} 
+
+`$PROJECT_SETTINGS` is the settings.xml with the repository which contains ${OPTAPLANNER_BUILD_VERSION} tar.gz 
```

Comment:
I think this applies to the original optaplanner-optawebs plugin only. Here we provide exact url to the tar.gz, but the settings are still needed for the dependency download and mainly proxy settings.
I would rephrase maybe similarly to:
`is the settings.xml with the repository which contains project dependencies.`

Reply:
yes, agree, will do it.